pkg/cmd/roachtest/run.go

```diff
@@ -327,3 +333,104 @@ func testRunnerLogger(
 	shout(ctx, l, os.Stdout, "test runner logs in: %s", runnerLogPath)
 	return l, teeOpt
 }
+
+// runOperations sequentially runs operations matched by the passed-in filter.
+func runOperations(register func(registry.Registry), filter string) error {
+	//lint:ignore SA1019 deprecated
+	rand.Seed(roachtestflags.GlobalSeed)
+	r := makeTestRegistry()
+
+	register(&r)
+	ctx := context.Background()
+	stopper := stop.NewStopper()
+	defer stopper.Stop(ctx)
+	// NB: root logger with no path always tees to Stdout.
+	l, err := logger.RootLogger("", logger.NoTee)
+	if err != nil {
+		return err
+	}
+	if len(roachtestflags.ClusterNames) == 0 {
+		return errors.New("roachtest: run-operation must be called with --cluster")
+	}
+	clusterName := roachtestflags.ClusterNames
```

Comment:
This can be a comma separated list of clusters. It might be better to use an arg or different flag to specify this. Although I would argue for an arg as a cluster is required.

Reply:
Changing to a (positional) arg, good point.